frontend/src/scss/global.scss

```diff
@@ -61,6 +61,13 @@ body {font-feature-settings: "ss03", "calt", "rlig", "kern", "cv05", "cv07", "cv
 	}
 }
 
+.v-list-item__title {
+	font-size: 16px;
+}
+.v-list-item__content {
+	align-content: flex-start;
+}
```

Comment:
The font establishes a better visual hierarchy.

The alignment of vuetify is in my opinion incorrect and looks off. This because we don't know how many lines the description has. When we set `two-line` instead of `three-line` the text gets cut off, which we don't want.  But the default behaviour of three-line requires the content to have 3 lines, which we don't know. 

**Default style:**
<img width="851" alt="Screenshot 2021-04-17 at 14 38 24" src="https://user-images.githubusercontent.com/3001985/115113506-09e1de00-9f8b-11eb-8373-5da062ab9dc7.png">

**With my changes:**
<img width="856" alt="Screenshot 2021-04-17 at 14 38 30" src="https://user-images.githubusercontent.com/3001985/115113516-1d8d4480-9f8b-11eb-9004-f8eb9e2e1df4.png">